src/refactor_nrepl/extract_definition.clj

```diff
@@ -8,7 +8,7 @@
   (:import [java.io PushbackReader StringReader]
            java.util.regex.Pattern))
 
-(defn- occurrence-to-map
+(defn occurrence-to-map
```

Comment:
Btw, why did you make this public?

Reply:
Because of this erorr `warning: Unused private var refactor-nrepl.extract-definition/occurrence-to-map`, maybe add this to kondo config instead?

Reply:
If it's really not used anywhere, I guess you can simply remove it.

Reply:
Yes, that's the idea :)